packages/backend/src/graphql/resolvers/fee.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  ResolversTypes,
+  MutationResolvers,
+  Fee as SchemaFee
+} from '../generated/graphql'
+import { ApolloContext } from '../../app'
+import { isFeeError, errorToCode, errorToMessage } from '../../fee/errors'
+import { Fee } from '../../fee/model'
+
+export const updateFee: MutationResolvers<ApolloContext>['updateFee'] = async (
+  parent,
+  args,
+  ctx
+): Promise<ResolversTypes['UpdateFeeResponse']> => {
+  const feeService = await ctx.container.use('feeService')
+  try {
+    const feeOrError = await feeService.create(args.input)
+
+    if (isFeeError(feeOrError)) {
+      return {
+        code: errorToCode[feeOrError].toString(),
+        success: false,
+        message: errorToMessage[feeOrError]
+      }
+    }
+    return {
+      code: '200',
+      success: true,
+      message: 'Fee updated',
+      fee: feeToGraphql(feeOrError)
+    }
+  } catch (error) {
+    ctx.logger.error(
+      {
+        options: args.input,
+        error
+      },
+      'error updating fee'
+    )
+    return {
+      code: '500',
+      success: false,
+      message: 'Error trying to update fee'
+    }
+  }
+}
+
+export const feeToGraphql = (fee: Fee): SchemaFee => {
+  return {
+    id: fee.id,
+    assetId: fee.assetId,
+    type: fee.type,
+    fixed: fee.fixedFee,
+    percentage: parseFloat(fee.percentageFee),
```

Comment:
we need to check whether `fee.percentageFee` is defined or not, otherwise I get an error.

Reply:
Correct me if I'm wrong but you ran into this when you created a fee without the percentage fee (when both fees were optional). There shouldnt be a problem here anymore because we're requiring both fees, including at the db level, so this will never be undefined.